bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -4261,4 +4263,32 @@ public boolean getHighlightEnabled() {
 	checkWidget();
 	return highlightEnabled;
 }
+/**
+ * Update the cached minimize button image when color is changed.
+ */
+private void updateMinImage() {
+	if (showMin)	{
+		if (minMaxTb != null)	{
+			if (minItem != null)	{
```

Comment:
Is there any special reason for three levels of nesting when && could be used instead?

Reply:
> Is there any special reason for three levels of nesting when && could be used instead?

Coded the issue logically but dint think of writing it in minimal lines, Thanks for your observation. Done.